src/seedu/addressbook/AddressBook.java

```diff
@@ -606,6 +615,26 @@ private static String executeListAllPersonsInAddressBook() {
     }
 
     /**
+     * Displays all persons in the address book to the user; in alphabetical order.
+     *
+     * @return feedback display message for the operation result
+     */
+    private static String executeListAllPersonsInAddressBookSorted() {
```

Comment:
errr... the method name suggests you are listing the people in the addressbook, however, the functionality includes sorting as well. clearly this is violation of SLAP.
please extract out the sort function to a separate method and then invoke that here.